tuxemon/technique.py

```diff
@@ -272,7 +287,34 @@ def use(self, user: Monster, target: Monster) -> TechniqueResult:
         self.next_use = self.recharge_length
 
         return meta_result
-
+    
+    def changestats(self, user: Monster, target: Monster) -> EffectResult:
+        ''' JSON SYNTAX:
+        Value: the number to change the stat by, default is add, use negative to subtract.
+        Dividing: set this to True to divide instead of adding or subtracting the value.
+        Overridetofull: in most cases you wont need this, if True it will set the stat to its original value rather than the specified value, but due to the way the 
+        game is coded, it currently only works on hp.
+        '''
+        statsmaster = [self.statspeed, self.stathp, self.statarmour, self.statmelee, self.statranged, self.statdodge]
+        statslugs = ['speed', 'hp', 'armour', 'melee', 'ranged', 'dodge']
+        for stat, slug in zip(statsmaster, statslugs):
+            if not stat:
```

Comment:
this is a bug.  my previous suggestion was
```python
if not stat:
    continue
```
right now, if stat is None or an empty list, there will be an error when it is used as a dictionary.  please fix this and play test the game.

Reply:
Ok, sorry thought the continue was a placeholder. I'll fix that.